recipe/meta.yaml

```diff
@@ -3,7 +3,7 @@
 # Copyright (c) 2017, Continuum Analytics, Inc. All rights reserved.
 #
 # Distribution of the content approved by NVIDIA ( http://nvbugs/3052604 )
-{% set major_minor = "9.2" %}
+{% set major_minor = "11.1" %}
```

Comment:
I believe this should be `11.1` LMK if not

Reply:
Yeah sorry this is leftover from other changes. Fixed with PR ( https://github.com/conda-forge/cudatoolkit-feedstock/pull/24 ). Integrated that change into this PR as well. So this should be fixed.